test/Makefile

```diff
@@ -4,7 +4,7 @@ install-kind:
 
 .PHONY: shard-%
 shard-%:
-	./scripts/e2e-kind.sh $@
+	./scripts/e2e-kind.sh $@ $(filter-out $@,$(MAKECMDGOALS)) 
```

Comment:
I can't decide if this is clever or evil

most other places in k8s/openshift would do something like `make shard-test WHAT="should enforce egress policy allowing traffic to a server in a different namespace based on PodSelector and NamespaceSelector"`